en/docs/tutorials/construct-statements.md

```diff
@@ -0,0 +1,119 @@
+
+    ![Edit variable expression](../assets/img/statement-editor/edit-variable-expression.gif){.cInlineImage-half}
+
+    Here, you are entering static text as a part of the variable value.
+
+8. Click **Expressions**, and under the **Arithmetic** section, click the **Es + Ex** expression.
+
+    !!! info
+        The **Expressions** tab lists all the pre-defined templates that you can use to define complex expressions.
+
+    ![Select expression template](../assets/img/statement-editor/select-expression-template.gif){.cInlineImage-half}
+
+    In the `Es + Ex` expression template, `Es` refers to the existing expression you clicked and selected. `+ Ex` combines another expression with `Es`.
+
+9. Click **Libraries** and search for `max`. In the filtered list, click  **lang.int:max**.
+
+    !!! info
+        The **Libraries** tab lists all the Ballerina Standard and Language libraries.
+
+    ![Select library](../assets/img/statement-editor/select-library.gif){.cInlineImage-half}
+
+    `lang.int:max` is a Ballerina standard library for deriving the maximum value.
+
+     You will see some diagnostics generated for the library you selected. Move on to the next step to resolve them. 
+
+10. Click the **int0:max(n,ns)** function. 
+
+     When you select a function or a method call, the **Parameters** tab opens by default. This tab displays information about the selected function or method call, including details about parameters, descriptions, and return type descriptions. In this tab, you can easily add parameters to the expression by selecting the check boxes for the relevant parameters.
+
+     ![Select function](../assets/img/statement-editor/select-function.gif){.cInlineImage-half}
+
+     !!! note
+         When a parameter requires a function or a method call to work, the Statement Editor selects its check box for that parameter by default and does not allow you to clear it.
+
+11. Change the **n** and **ns** parameter values to integers. To make each parameter value editable, you can double-click on it. Let's change them as follows:
+
+     | **Parameter** | **Value** |
+     |---------------|-----------|
+     | **n**         | **`30`**  |
+     | **ns**        | **`-89`** |
+
+12. Select **int0:max(30,-89)** and click **Suggestions**. Click **toString()** in the suggestions list to convert the value that the expression derived to a string format.
+
+     ![Convert to string](../assets/img/statement-editor/convert-to-string.gif){.cInlineImage-half}
+
+     Note that the Statement Editor has cleared all the diagnostics messages, indicating that the statement is valid.
+
+     You will see the variable statement to be as follows:
+
+     ```
+     string maxValue = "Max : " + int0:max(30, -89).toString();
+     ```
+    
+Congratulations! You have constructed a valid statement via the Statement Editor!
+
+You can click **Save** and add the variable statement to the low-code diagram.
+
+!!! tip
```

Comment:
Should we move this tip to the top so that it will be useful to a user who would want to undo, redo, or delete a step while constructing the statement?

Reply:
Done